docs/isp_flashing_guide.md

```diff
@@ -48,53 +102,69 @@ This is pretty straight-forward - we'll be connecting like-things to like-things
+You can find the stock bootloaders in the [`util/` folder](https://github.com/qmk/qmk_firmware/tree/master/util). Be sure to flash the correct bootloader for your chip:
 
 * [`atmega32u4`](https://github.com/qmk/qmk_firmware/blob/master/util/bootloader_atmega32u4_1_0_0.hex) - Most keyboards, Planck Rev 1-5, Preonic Rev 1-2
+8 [`Pro Micro`](https://github.com/adafruit/Caterina-Bootloader/blob/master/Built%20Firmwares/Caterina-Leonardo.hex) - The default bootloader for Pro Micro controllers
 * [`at90usb1286`](https://github.com/qmk/qmk_firmware/blob/master/util/bootloader_at90usb128x_1_0_1.hex) - Planck Light Rev 1
-* [`atmega32a`](https://github.com/qmk/qmk_firmware/blob/master/util/bootloader_atmega32a_1_0_0.hex) - jj40
+* [`atmega32a`](https://github.com/qmk/qmk_firmware/blob/master/util/bootloader_atmega32a_1_0_0.hex) - jj40, and other V-USB/ps2avrGB keyboards
 
 If you're not sure what your board uses, look in the `rules.mk` file for the keyboard in QMK. The `MCU =` line will have the value you need. It may differ between different versions of the board.
 
-### Advanced/Production Techniques
+### Production Techniques
 
-If you'd like to flash both the bootloader **and** the regular firmware at the same time, you need to combine the files. 
+If you'd like to flash both the bootloader **and** the regular firmware at the same time, there are two options to do so.  Manually, or with the `:production` target when compiling. 
+
+To do this manually: 
 
 1. Open the original firmware .hex file in a text editor
 2. Remove the last line (which should be `:00000001FF` - this is an EOF message)
 3. Copy the entire bootloader's contents onto a new line (with no empty lines between) and paste it at the end of the original file
 4. Save it as a new file by naming it `<keyboard>_<keymap>_production.hex`
 
-It's possible to use other bootloaders here in the same way, but __you need a bootloader__, otherwise you'll have to use ISP again to write new firmware to your keyboard.
+?> It's possible to use other bootloaders here in the same way, but __you need a bootloader__, otherwise you'll have to use ISP again to write new firmware to your keyboard.
+
+To do this the easy way, you can flash the board using the `:production` target when compiling.  This compiles the firmware, then compiles the QMK DFU bootloader, and then creates a combined image.  Once this is done, you'll see three files: 
+* `<keyboard>_<keymap>.hex`
+* `<keyboard>_<keymap>_bootloader.hex`
+* `<keyboard>_<keymap>_production.hex`
+
+The QMK DFU bootloader has only really been tested on `atmega32u4` controllers (such as the Planck boards, and the Pro Micro), and hasn't been tested on other controllers.  However, it will not work on V-USB controllers, such as the `atmega32a` or `atmega32p`. 
```

Comment:
```suggestion
The QMK DFU bootloader has only really been tested on `atmega32u4` controllers (such as the Planck boards, and the Pro Micro), and hasn't been tested on other controllers.  However, it will not work on V-USB controllers, such as the `atmega32a` or `atmega328p`. 
```